app/ErrorHandler.scala

```diff
@@ -0,0 +1,40 @@
+import javax.inject.Singleton
+
+import controllers.routes
+import play.api.http.HttpErrorHandler
+import play.api.mvc.Results._
+import play.api.mvc.{RequestHeader, Result}
+
+import scala.concurrent.Future
+
+@Singleton
+class ErrorHandler extends HttpErrorHandler {
+
+  val NOT_FOUND = 404
+  val BAD_REQUEST = 400
+  val FORBIDDEN = 403
+  val PROXY_AUTHENTICATION_REQUIRED = 407
+
+  override def onClientError(request: RequestHeader, statusCode: Int, message: String): Future[Result] = {
+
+    statusCode match {
+
+      case NOT_FOUND                     => Future.successful(Redirect(routes.SessionsController.sessions(1, None)).flashing("error" -> "Page not found!"))
+      case BAD_REQUEST                   => Future.successful(Redirect(routes.SessionsController.sessions(1, None)).flashing("error" -> "Bad Request!"))
```

Comment:
Please change "Bad Request!" to "Bad request!", i.e change 'R' to 'r' as used in all other flash messages in this file.